ironfish/src/testUtilities/nodeTest.ts

```diff
@@ -74,7 +74,7 @@ export class NodeTest {
 
     const sdk = await IronfishSdk.init({ dataDir, strategyClass })
 
-    sdk.config.setOverride('bootstrapNodes', [''])
```

Comment:
We might want to keep the bootstrap nodes empty here -- I haven't debugged into it to see if it actually happens, but the tests might try to connect to the bootstrap node otherwise

Reply:
Added back